chapter_recurrent-modern/lstm_vn.md

```diff
@@ -47,7 +47,7 @@ Chúng ta hãy xem thiết kế này hoạt động như thế nào trong thực
 ## Gated Memory Cells
 -->
 
-## Các Ô Nhớ có Cổng
+## Ô nhớ được kiểm soát
```

Comment:
mình thấy từ cũ nghe cũng ổn mà

Reply:
Yeah, mình nên bàn thêm chỗ này vì theo mình từ `gated` này là giống như `canh gác` thay vì là `cổng`

Reply:
triệu hồi @lkhphuc

Reply:
Mình nghiêng về cách dịch `có cổng` cho thống nhất với GRU cũng như ở bên dứoi.

Reply:
@lkhphuc cho em hỏi tại sao lúc đầu mình dịch là `có cổng` vậy? Em không kiếm được phần thảo luận

Reply:
Từ này lúc thêm vô không có thảo luận, mọi người đều đông ý hết @rootonchair .
Lý do một unit bị `gated` là dó nó có `gates`, `gates` thì đúng là để kiểm soát nên dịch là `được kiểm soát` hoàn toàn đúng. Còn việc `gated` dịch là `có cổng` hay `được kiểm soát` mình theo phương án `có cổng` hơn.
Cộng thêm việc mình thích giữ lại các hình ảnh có trong tiếng anh, nếu chỉ muốn nhấn mạnh việc `được kiểm soát` thì bản gốc cũng có thể dùng `Controlled Memory Cells` chẳng hạn.